### ISSUE TITLE
Funded fuzzing running out

### ISSUE THREAD

**Issue by lukaslueg** [kdave/btrfs-progs#56]
Hi,

there are still several weeks worth of donated 24/7-fuzzing available. The grace period on that funding is running out in October, unused funds expire by then. Most of the bugs [found so far](https://bugzilla.kernel.org/buglist.cgi?bug_status=NEW&bug_status=ASSIGNED&bug_status=REOPENED&component=btrfs&email2=lukas.lueg%40gmail.com&emailassigned_to2=1&emailcc2=1&emailreporter2=1&emailtype2=equals&query_format=advanced&short_desc=fuzzed&short_desc_type=allwordssubstr) are currently unresolved and there has been no movement in close to a year.

In order not to swamp reports, the above-mentioned bugs need to be resolved before we can continue fuzzing BTRFS. Is there a chance for progress on this? If not, I'd use those funds for other OSS-work.

**Comment by adam900710** [comment]
I'll take them.

A quick glance shows that some of them are not hard to handle.

**Comment by adam900710** [comment]
https://bugzilla.kernel.org/show_bug.cgi?id=172861
https://bugzilla.kernel.org/show_bug.cgi?id=172811
https://bugzilla.kernel.org/show_bug.cgi?id=168301
https://bugzilla.kernel.org/show_bug.cgi?id=167921
https://bugzilla.kernel.org/show_bug.cgi?id=167551
https://bugzilla.kernel.org/show_bug.cgi?id=161821
https://bugzilla.kernel.org/show_bug.cgi?id=161811
https://bugzilla.kernel.org/show_bug.cgi?id=156741
https://bugzilla.kernel.org/show_bug.cgi?id=156731

Those images are merged into fuzzed tests, and IIRC such merged cases normally means they are fixed in btrfs-progs.
It's just the BZ not closing.

**Comment by kdave** [comment]
On Mon, Sep 18, 2017 at 07:52:27AM +0000, Qu Wenruo wrote:
> Those images are merged into fuzzed tests, and IIRC such merged cases normally means they are fixed in btrfs-progs.
> It's just the BZ not closing.

The images were merged if I was able to fix the bug, but some of the
bugs turned out to require broader changes. I've merged most of them so
we have them in the repository. This gives it more exposure to
developers who don't read bugzilla but run the tests. At the cost that
test-fuzz still fails.
